pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1023,7 +1023,9 @@ func (g *genericScheduler) selectNodesForPreemption(
 		if meta != nil {
 			metaCopy = meta.ShallowCopy()
 		}
-		pods, numPDBViolations, fits := g.selectVictimsOnNode(pluginContext, pod, metaCopy, nodeNameToInfo[nodeName], fitPredicates, queue, pdbs)
+		pluginContextClone := pluginContext.Clone()
```

Comment:
why not do the copy in `selectVictimsNode`, given that pluginContextClone isn't used outside of it?

Also, prefer shorter names for local variables. We could start using `pCtx` instead of `pluginContext`

Reply:
> why not do the copy in `selectVictimsNode`, given that pluginContextClone isn't used outside of it?
> 

To be consistent with how "meta" is treated.

> Also, prefer shorter names for local variables. We could start using `pCtx` instead of `pluginContext`

we already use pluginContext everywhere.

Reply:
sg, they are just nits.